src/Host/Client/Impl/DataInspection/IRActiveBindingInfo.cs

```diff
@@ -7,8 +7,9 @@ namespace Microsoft.R.DataInspection {
     /// </summary>
     public interface IRActiveBindingInfo : IREvaluationResultInfo {
         /// <summary>
-        /// Value of the active binding after evaluation.
+        /// This is null if evaluate active bindings in RTools Options is set 
```

Comment:
The doc comment should be referencing a lower-level concept, i.e. `REvaluationResultProperties.ComputedValueProperty` in this case, since this layer is not supposed to be VS-dependent or UI-dependent in general (and will likely be shared with VSCode in the future).